google/cloud/bigtable/examples/bigtable_grpc_credentials.cc

```diff
@@ -23,34 +22,20 @@ struct Usage {
   std::string msg;
 };
 
-char const* ConsumeArg(int& argc, char* argv[]) {
-  if (argc < 2) {
-    return nullptr;
-  }
-  char const* result = argv[1];
-  std::copy(argv + 2, argv + argc, argv + 1);
-  argc--;
-  return result;
-}
-
 std::string command_usage;
 
-void PrintUsage(int, char* argv[], std::string const& msg) {
-  std::string const cmd = argv[0];
+void PrintUsage(std::string const& cmd, std::string const& msg) {
   auto last_slash = std::string(cmd).find_last_of('/');
```

Comment:
Opt: The `std::string()` around `cmd` looks like a confusing remnant.

Reply:
Fixed.